### PR TITLE
fix: check styleResources for existence

### PR DESCRIPTION
**Pull request by aldarund**
## Types of changes

- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description
Check styleResource for its existence, so it dont fail silently without giving clue to user why its not working.

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [x] I have added tests to cover my changes (if not applicable, please state why)
- [x] All new and existing tests are passing.

### CODE REVIEW

File: packages/builder/src/builder.js [hunk 1]
```diff
@@ -497,6 +498,18 @@ export default class Builder {
       }
     }))
 
+    // Check styleResource exist
```
Comment (manniL) [inline packages/builder/src/builder.js]: Maybe [`options.js`](https://github.com/nuxt/nuxt.js/blob/dev/packages/common/src/options.js) is a better place for the check?

**Review by TheAlexLichter — CHANGES_REQUESTED** [review]

File: packages/builder/src/builder.js (hunk 1 above)
Comment (aldarund) [reply packages/builder/src/builder.js]: Maybe. But why check for plugins.js is here and not in options then  ? :)

Comment (pi0) [reply packages/builder/src/builder.js]: Wecare trying to scan file system which is only specific to builder ( l mean not required for production nuxt start) So here is probably better place and helps decoupling packages.

**Review by TheAlexLichter — DISMISSED** [review]

**Review by pi0 — DISMISSED** [review]

**Review by galvez — CHANGES_REQUESTED** [review]
@aldarund 

```
$ ack "throw new Error" | grep -i not
builder/src/builder.js:445:          throw new Error(`Could not compile template ${src}: ${err.message}`)
builder/src/builder.js:478:        throw new Error(`Plugin not found: ${p.src}`)
core/src/module.js:41:      throw new Error('Template src not found:' + src)
core/src/resolver.js:86:    throw new Error(`Cannot resolve "${path}" from "${resolvedPath}"`)
```

To maintain consistency, can you change it to read `Style src not found:`?

**Comment by aldarund** [comment]
@galvez not quite follow you. Style != Style Resource. Style would be misleading because it could be confused with css section. And it same style message as with plugin not found. And its not quite src because it could be glob pattern

**Review by galvez — DISMISSED** [review]

**Comment by manniL** [comment]
Conflicts again 🙈

**Review by pi0 — APPROVED** [review]

File: packages/webpack/src/builder.js [hunk 2]
```diff
@@ -60,6 +65,18 @@ export default class WebpackBuilder {
       }
     }
 
+    // Check styleResource existence
+    const styleResources = this.context.options.build.styleResources
```
Comment (pi0) [inline packages/webpack/src/builder.js]: @clarkdo As this was a loader, I've just kept in the place as original PR was. What do you think?

**Pull request merged**